examples/nthash_spacedseeds.cpp

```diff
@@ -0,0 +1,20 @@
+#include "../include/btllib/nthash.hpp"
+#include "../include/btllib/seq_reader.hpp"
```

Comment:
SeqReader is not used in the example, so this `#include` can be removed. Since strings and vectors are used, you should `#include <vector>` and `#include <string>`

Reply:
Fixed in https://github.com/bcgsc/btllib/commit/32b76cb36cced8d838d5c5368616a3af05035ed0